packages/create-typink/src/utils/render.ts

```diff
@@ -18,6 +18,7 @@ export function renderHelpMessage() {
   console.log(` ${chalk.bold.blue('Options:')}
     ${chalk.gray('-n, --name')}                       Project name
     ${chalk.gray('-t, --template')}                   Template {inkVersion}-{walletConnector}-{ui} [inkVersion: <legacy|v6>, walletConnector: <typink|subconnectv2|talisman>, ui: <nextjs|vite>] (e.g. legacy-typink-vite) 
```

Comment:
We should generally improve this, the instruction here is a bit confusing for people. Maybe simplify this and have a list of template name below for people to pick.